regenerate.sh

```diff
@@ -17,7 +17,13 @@ tv_scripts=(
     sapling_key_components
     sapling_note_encryption
     sapling_signatures
-    sapling_zip32)
+    sapling_zip32
+    unified_address
+    unified_full_viewing_keys
+    zip_0143
+    zip_0243
+    zip_0244
+    )
```

Comment:
This will break the script (in a way that isn't detected until the script ends), because it inserts an empty-string `""` element at the end of the array. Remove the extra newline:
```suggestion
    zip_0244)
```